src/main/resources/assets/minecolonies/lang/manual_en_us.json

```diff
@@ -2127,5 +2127,94 @@
+  "minecolonies:sifter.job.desc": "Sifts through a variety of materials to find goodies.",
+  "minecolonies:florist.job.desc": "Plants a large variety of flowers for the colony.",
+  "minecolonies:enchanter.job.desc": "Gathers mana by observing others. Enchants colonist tools and creates enchanting books.",
+  "minecolonies:researcher.job.desc": "Researches the topics the player selects.",
+  "minecolonies:healer.job.desc": "Heals wounded entities and provides cures for the sick.",
+  "minecolonies:pupil.job.desc": "Sits around and learns from the material provided by the teacher.",
+  "minecolonies:teacher.job.desc": "Sits with the kids and provides material.",
+  "minecolonies:dyer.job.desc": "Handles most of the dyeing that is required in the colony.",
+  "minecolonies:glassblower.job.desc": "Provides most glass-based items.",
+  "minecolonies:mechanic.job.desc": "Handles redstone recipes and things other workers can't handle.",
+  "minecolonies:fletcher.job.desc": "Handles leather, string and wool based recipes.",
+  "minecolonies:planter.job.desc": "Provides sugar cane, cactus, and bamboo to the colony.",
+  "minecolonies:rabbitherder.job.desc": "Handles a fluffle of rabbits.",
+  "minecolonies:concretemixer.job.desc": "Creates concrete powder and actual concrete.",
+  "minecolonies:beekeeper.job.desc": "Cares for several beehives.",
+  "minecolonies:cookassistant.job.desc": "Handles food-based recipes the main cook can't handle.",
+  "minecolonies:smelter.job.desc": "Breaks ore blocks into raw ore and smelts raw ores into ingots.",
+  "minecolonies:druid.job.desc": "Support type unit. Buffs friendlies and nerfs enemies.",
+
+  "minecolonies:builder.skills.desc": "Influence the placement and breaking speed.",
+  "minecolonies:deliveryman.skills.desc": "Influences the walking speed and carrying capacity.",
+  "minecolonies:miner.skills.desc": "Influence the placement and breaking speed.",
+  "minecolonies:quarrier.skills.desc": "Influence the placement and breaking speed.",
+  "minecolonies:lumberjack.skills.desc": "Influences the breaking and walking speed.",
+  "minecolonies:farmer.skills.desc": "Influences the speed at which the farmer works.",
+  "minecolonies:undertaker.skills.desc": "Influences the working speed and resurrection chance.",
+  "minecolonies:fisherman.skills.desc": "Influences the fishing speed and loot.",
+  "minecolonies:baker.skills.desc": "Influences the recipe improvement chance and crafting speed",
+  "minecolonies:cook.skills.desc": "Influences the cooking speed.",
+  "minecolonies:shepherd.skills.desc": "Influences the amount of wool they obtain and the damage they deal.",
+  "minecolonies:cowboy.skills.desc": "Influences their working speed and the damage they deal.",
+  "minecolonies:swineherder.skills.desc": "Influences their working speed and the damage they deal.",
+  "minecolonies:chickenherder.skills.desc": "Influences the chance they succesfully pickup chicken byproducts and the damage they deal.",
```

Comment:
Wiki specifies greater chance to hit chickens, not more damage (though essentially the same result)